pkgs/development/tools/racerd/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, rustPlatform, makeWrapper }:
+
+with rustPlatform;
+
+buildRustPackage rec {
+  version = "0.1.1";
+  name = "racerd-${version}";
+
+  src = fetchFromGitHub {
+    owner = "jwilm";
+    repo = "racerd";
+    rev = "9b63607132ab7f9aa32eb85eb45707c1b8f70990";
+    sha256 = "0kggikin8hyq71hwbapczyvik39wwycvgdxigzhgivlcpss2wmdx";
+  };
+
+  depsSha256 = "";
```

Comment:
How do I get this and where do I get this from?

I cannot find documentation on this, neither in the nixpkgs repo nor in the wiki.